### PR TITLE
Hacktoberfest 2022 Contribution

### PR DESCRIPTION
**Pull request by ShehrozIrfan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (ShehrozIrfan) [inline index.html:3]: remove this extra line.

File: index.html [hunk 2]
```diff
@@ -8,14 +9,19 @@
...
+    integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
   <!-- google fonts: Roboto -->
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
   <!-- font awesome cdn -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <!-- Favicon -->
+  <link rel="icon" type="image/x-icon" href="./assets/images/favicon.png">
+
```
Comment (ShehrozIrfan) [inline index.html:22]: remove this extra line.

File: index.html [hunk 3]
```diff
@@ -8,14 +9,19 @@
...
 </head>
+
```
Comment (ShehrozIrfan) [inline index.html:24]: remove this extra line.

File: index.html [hunk 4]
```diff
@@ -26,48 +32,78 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
...
             </div>
           </div>
         </div>
-        <div class="col-md-4">
-          <div class="profile-section">
-            <div class="text-center">
-              <img src="./assets/images/uttam.jpg" alt="Uttam" class="profile-image">
-              <hr>
+      </div>
+
```
Comment (ShehrozIrfan) [inline index.html:58]: remove these extra line from 57-58

File: index.html [hunk 5]
```diff
@@ -26,48 +32,78 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
...
-              <div class="profile-github pt-2">
-                <p>
-                  <a href="https://github.com/uttam4124" target="_blank"><i class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
-                  <a href="https://www.linkedin.com/in/uttam-pandey-6346301aa/" target="_blank"><i class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i></a>
-                </p>
-              </div>
+          </div>
+        </div>
+      </div>
+
```
Comment (ShehrozIrfan) [inline index.html:81]: remove extra lines from 81-82

File: index.html [hunk 6]
```diff
@@ -26,48 +32,78 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
...
+                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
+                <a href="https://www.linkedin.com/in/shaikh-minhaj-softwareengineer/" target="_blank"><i
+                    class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i></a>
+              </p>
             </div>
           </div>
         </div>
-       
+      </div>
+
```
Comment (ShehrozIrfan) [inline index.html:104]: remove extra line.

File: index.html [hunk 7]
```diff
@@ -26,48 +32,78 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
...
     </div>
   </div>
 </body>
-</html>
+
```
Comment (ShehrozIrfan) [inline index.html:108]: remove extra line.

**Comment by ShehrozIrfan** [comment]
@minhaj-313 
Excellent work!
Remove the extra lines and then push again.

**Comment by ShehrozIrfan** [comment]
@minhaj-313 
Excellent Job. PR is merged.
You can see the results on [Github Pages](https://shehrozirfan.github.io/hacktoberfest-2022-first-contribution/)

**Comment by minhaj-313** [comment]
> @minhaj-313 Excellent Job. PR is merged. You can see the results on [Github Pages](https://shehrozirfan.github.io/hacktoberfest-2022-first-contribution/)

Thank you So Much , Please label as hacktoberfest-accepted

**Comment by ShehrozIrfan** [comment]
@minhaj-313 
You can follow me and star this repo for further updates.
Thanks. Have fun 👍

**Pull request merged**